app/assets/stylesheets/components/_navbar.scss

```diff
@@ -29,6 +45,6 @@
 }
 
 .bottom-nav-icon-show {
-  opacity: .7;
+  opacity: 1;
 }
```

Comment:
Originally, opacity for the nav-icons was 0 in the _.bottom-nav-icon class_ (aka invisible). When the avatar gets clicked the _.bottom-nav-icon-show_  class would be added to the icons, the  _opacity 0.7_   declaration would override the _opacity 0_ declaration, so the icons would become visible (check out _javascript/packs/show_bottom_nav.js_).  Now that opacity is 0.8 in the _.bottom-nav-icon_ class, the _.bottom-nav-icon-show_ class is not needed any more and can be deleted. The same goes for the JS code in _show_bottom_nav.js_.

Reply:
@Matth93 if we only have the .8 on the .bottom-nav-icon class, will the icon still be invisible before the avatar clicks their profile image?  I'm ok axing the .bottom-nav-icon-show if so.

Reply:
No, if  their opacity is 0 in  _.bottom-nav-icon class_  it will be invisible by default,  the JS method will add the _.bottom-nav-icon-show_ to the icon, overriding the 0 opacity and making it visible.

Reply:
Cool - I think we can change it back because it seemed my styling wasn't allowing the _.bottom-nav-icon-show_ to appear